src/data/CallAPI.jsx

```diff
@@ -52,10 +52,9 @@ export function RecipientsAPI(
     query = `${query}${id}/`;
   } else if (limit !== 9999 && offset != null) {
     query = `${query}?limit=${limit}&offset=${offset}`;
-  } else {
+  } else if (id == null && offset == null && limit === 9999) {
     query = `${query}?limit=${limit}`;
   }
```

Comment:
여기 조건이 겹쳐서 
```suggestion
if (id) {
    query = `${query}${id}/`;
  } else {
    query = `${query}?limit=${limit}`;
    if (limit !== 9999 && offset != null) {
    query = `${query}&offset=${offset}`;
  }
}
```
바구면 좋을거같아용